src/masternodes/govvariables/attributes.h

```diff
@@ -278,6 +280,60 @@ struct CDexTokenInfo {
 
 enum FeeDirValues : uint8_t { Both, In, Out };
 
+struct CTransferDomainAccounting {
+    CAccountingBalaces dvmEvmTotal;
+    CAccountingBalaces evmDvmTotal;
+    CAccountingBalaces dvmIn;
+    CAccountingBalaces evmIn;
+    CAccountingBalaces dvmOut;
+    CAccountingBalaces evmOut;
+    CAccountingBalaces dvmCurrent;
+    CAccountingBalaces evmCurrent;
+
+    ADD_SERIALIZE_METHODS;
+
+    template <typename Stream, typename Operation>
+    inline void SerializationOp(Stream &s, Operation ser_action) {
+        READWRITE(dvmEvmTotal);
+        READWRITE(evmDvmTotal);
+        READWRITE(dvmIn);
+        READWRITE(evmIn);
+        READWRITE(dvmOut);
+        READWRITE(evmOut);
+        READWRITE(dvmCurrent);
+        READWRITE(evmCurrent);
+    }
+};
+
+struct CEvmFees {
```

Comment:
I'd add the `From` and `SetToAttributes` pattern so usage is uncluttered without noise like the attribute keys.